cmd/az-analyze/.az-analyze.yml

```diff
@@ -0,0 +1,2 @@
+kubeconfig: ".kube/config"
```

Comment:
Do we need the '.' in the file name? Can we rename it to az-analyze.yml? It would be even better if we create a config folder and add this file under that.

Reply:
Removed "." in the file name and moved it to a new config folder.